lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
`use strict` also affects that behavior supposedly. But like I said, even if they're modified, nothing is using `arguments` so...

(maybe if someone used a breakpoint with a debugger they'd be able to access it?)